src/doc/book/macros.md

```diff
@@ -509,7 +509,7 @@ A macro defined within the body of a single `fn`, or anywhere else not at
 module scope, is visible only within that item.
 
 If a module has the `macro_use` attribute, its macros are also visible in its
-parent module after the child’s `mod` item. If the parent also has `macro_use`
+parent module after its `mod` item. If the parent also has `macro_use`
```

Comment:
I actually think this is worse. Not to say the wording was perfect before, but now it's not clear whether "its" refers to the parent module or the child module.